python/pyarrow/tests/test_convert_pandas.py

```diff
@@ -573,7 +579,7 @@ def test_decimal_128_from_pandas(self):
         })
         converted = pa.Table.from_pandas(expected)
         field = pa.field('decimals', pa.decimal(26, 11))
-        schema = pa.schema([field])
+        schema = pa.schema([field, DEFAULT_INDEX_FIELD])
```

Comment:
This `DEFAULT_INDEX_FIELD` is a slight nuisance. Perhaps add an argument to from_pandas whether to ingest the index (default could be True or False I guess)?